chanzuckerberg/idseq-web
# Fix existing water control on metadata update

# Description

Fixes issue where existing values of water control were set back to default on update of metadata. 

# Notes

* This was a side effect of fixing the bigger bug of default values on upload

# Tests

* Try editing project with existing water control, see "yes"
* Try uploading new samples, see "no" and not stopped by "required field" error
